Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -304,6 +305,8 @@ private void InitializeKeyCommands()
         public ResultsViewModel ContextMenu { get; private set; }
         public ResultsViewModel History { get; private set; }
 
+        public bool gameModeStatus = false;
```

Comment:
tip, bools are initialised as false, so you dont need to set it to false.
we can use a getter and setter here actually
```suggestion
        public bool GameModeStatus { get; set; }
```

Reply:
public variables use pascal case where as private variables are lower camel case.